src/Test/Behat/FactoriesContext.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Zenstruck\Foundry\Test\Behat;
+
+use Behat\Behat\Context\Context;
+use Psr\Container\ContainerInterface;
+use Zenstruck\Foundry\ChainManagerRegistry;
+use Zenstruck\Foundry\Factory;
+use Zenstruck\Foundry\Test\LazyManagerRegistry;
+use Zenstruck\Foundry\Test\TestState;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class FactoriesContext implements Context
+{
+    public function __construct(private ContainerInterface $container)
+    {
+    }
+
+    /**
+     * @BeforeScenario
```

Comment:
These annotations cause race conditions because you cannot guarantee when they are run... hence why I think subscriber is the way to go.